### PR TITLE
add missing translation

### PR DESCRIPTION
**Pull request by infabo**
for messages in AccountController->_welcomeCustomer method

### CODE REVIEW

File: app/locale/de_DE/Mage_Customer.csv [hunk 1]
```diff
@@ -28,7 +28,9 @@
 "Admin","Admin"
 "All","Alle"
 "All Store Views","Alle StoreViews"
-"All countries","Alle Länder"
+"All countries","Allef you are a registered VAT Länder"
```
Comment (riker09) [inline app/locale/de_DE/Mage_Customer.csv]: Uh, what now? Looks like a search & replace error.

Comment (Schrank) [inline app/locale/de_DE/Mage_Customer.csv]: you already deleted your repo? Then I would say, fork again, fix again, PR again :-/

Comment (riker09) [inline app/locale/de_DE/Mage_Customer.csv]: This wasn't done by me. Original author is @infabo

**Comment by Schrank** [comment]
Yes, although it is configurable(?) which address is used for VAT calculation? so this should be defined by the configuration :-/

File: app/locale/de_DE/Mage_Customer.csv (hunk 1 above)
Comment (infabo) [inline app/locale/de_DE/Mage_Customer.csv:31]: No, I will resubmit the PR. I deleted the fork and GitHub now displays some random whatever :D

**Pull request closed without merging**